organizations/admin.py

```diff
@@ -12,8 +12,8 @@ class OrganizationAdmin(admin.ModelAdmin):
     Admin for the Organization table.
     soft-delete on the organizations
     """
-    list_display = ('name', 'description', 'created', 'active')
-    fields = ('name', 'description', 'created', 'active')
+    list_display = ('name', 'description', 'logo', 'created', 'active')
+    fields = ('name', 'description', 'logo', 'created', 'active')
```

Comment:
Actually, I just added it because of modified field was visible on the admin form (inherited from TimeStampedModel) while creating an organization.